tests/sentry/api/endpoints/test_organization_member_team_details.py

```diff
@@ -72,7 +75,8 @@ def owner_on_team(self):
 
     @cached_property
     def team_admin(self):
-        member = self.create_member(organization=self.org, user=self.create_user(), role="member")
+        self.team_admin_user = self.create_user()
```

Comment:
What's the difference between a `team_admin_user` and an `admin_on_team_user`? Could we add a comment somewhere mentioning this?

Reply:
`Team Admin` is a team-scoped role, while `Admin` is an org-scoped role. We've deprecated org `Admin` on business plans and replaced it with `Team Admin` to be able to give more granular permissions to people. `team_admin_user` is the user for a team admin on a team, while `admin_on_team` is an org admin who is also on the team.